matchms/importing/load_from_json.py

```diff
@@ -112,3 +112,12 @@ def parse_fieldname(key):
                         metadata_harmonization=metadata_harmonization)
     logger.info("Empty spectrum found (no peaks in 'peaks_json'). Will not be imported.")
     return None
+
+
+def scores_json_decoder(dct):
+    """
+    Object_hook function to convert JSON dictionary with :py:class:`~matchms.Score.Score` object into a python dictionary.
+    """
+    if "__Scores__" not in dct and "__Similarity__" not in dct:
+        return dict2spectrum(dct, metadata_harmonization=False)
+    return dct
```

Comment:
I think `dct` is a parameter name that is too error prone!

Reply:
The Python's documentation uses that name in the example, so it didn't occur to me as erroneous:
https://docs.python.org/3/library/json.html?highlight=json#module-json
```python
import json
def as_complex(dct):
    if '__complex__' in dct:
        return complex(dct['real'], dct['imag'])
    return dct
```

Reply:
Oh wow :D Different people have different opinions - but okay, then this seems to be generally accepted.

Reply:
I get the point. I'd probably come up with another name hadn't I seen this example.